daycare/migrations/0004_alter_image_daycare_id.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.3 on 2022-04-06 20:54
+
```

Comment:
Is it possible to merge migrations 3 and 4?

Reply:
it was generated by Django when running makemigrations, I can make the migrations again to handle that.